README.md

```diff
@@ -158,9 +165,11 @@ Render a form that will be automatically posted when the U2F device reponds.
 
 ```javascript
 // render requests from server into Javascript format
-var signRequests = <%= @sign_requests.as_json.to_json.html_safe %>;
+var signRequests = <%= @sign_requests.to_json.html_safe %>;
+var challenge = #{@challenge.to_json.html_safe};
+var appId = #{@app_id.to_json.html_safe};
```

Comment:
Corrected.  (I'd pasted those lines in from the HAML in the example, which, as before, uses #{...} syntax consistently.)